kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java

```diff
@@ -396,19 +397,36 @@ public final T createOrReplace(T... items) {
       return withName(itemToCreateOrReplace.getMetadata().getName()).createOrReplace(itemToCreateOrReplace);
     }
 
-    try {
-      // Create
-      KubernetesResourceUtil.setResourceVersion(itemToCreateOrReplace, null);
-      return create(itemToCreateOrReplace);
-    } catch (KubernetesClientException exception) {
-      if (exception.getCode() != HttpURLConnection.HTTP_CONFLICT) {
-        throw exception;
+    final CompletableFuture<T> future = new CompletableFuture<>();
+    while (!future.isDone()) {
```

Comment:
We can make the application go into an infinite loop in case some server error is permanently (or for a long period) affecting the server.
Maybe we should add a a MAX_RETRIES limit and check it here too.